tools/webinstall/install.sh

```diff
@@ -1113,7 +1113,9 @@ install_darwin() {
 
     need_cmd "$BREW"
 
-    do_cmd "$BREW" install unikraft/cli/kraftkit
+    _ind_cmd=$(printf "%s%s%s" "$BREW" " install" " unikraft/cli/kraftkit")
+
+    do_cmd "$_ind_cmd"
```

Comment:
Can this not be turned into a single string?
```bash
do_cmd "$BREW install unikraft/cli/kraftkit"
```
?

Reply:
I guess so, let me try

Reply:
yeah it works, whoops :D